### ISSUE TITLE
How is latestPluginVersion determined?

### ISSUE THREAD

**Issue by MelvinAchterhuis** [FriendsOfShopware/shopware-static-data#12]
### Question

![image](https://github.com/FriendsOfShopware/shopware-static-data/assets/26538915/2294754d-8147-471f-9ea6-d2579a9d5b78)

For 6.4 the latest version is 1.0.2.7, but for 6.5 the latest version is 2.0.2. 

1.0.27 is released after 2.0.2, so I guess the latest version is based upon that? 

But would be nice to either have both 6.4 and 6.5 in the file, or have a separate file for each major SW version.

**Comment by shyim** [comment]
yes, that's something we need to fix also in shopmon to take the correct one 😅
